add link for furder reading on evm (#10995)
ethereum/ethereum-org-website: src/content/developers/docs/evm/index.md

- [Jellopaper aka KEVM: Semantics of EVM in K](https://jellopaper.org/)
- [The Beigepaper](https://github.com/chronaeon/beigepaper)
- [Ethereum Virtual Machine Opcodes](https://www.ethervm.io/)

> - [Evm explaination for beginners](https://github.com/ethereumbook/ethereumbook/blob/develop/13evm.asciidoc))
> - [Ethereum Virtual Machine Opcodes Interactive Reference](https://www.evm.codes/)
> - [A short introduction in Solidity's documentation](https://docs.soliditylang.org/en/latest/introduction-to-smart-contracts.html#index-6)

Reviewer: Per guidelines, we should add listing to the end... Also, this links to the Mastering Ethereum book, so I would just use the name of the book and chapter as the label:

```suggestion
- [Ethereum Virtual Machine Opcodes Interactive Reference](https://www.evm.codes/)
- [A short introduction in Solidity's documentation](https://docs.soliditylang.org/en/latest/introduction-to-smart-contracts.html#index-6)
- [Mastering Ethereum - The Ethereum Virtual Machine](https://github.com/ethereumbook/ethereumbook/blob/develop/13evm.asciidoc))
```
